replica/table.cc

```diff
@@ -957,7 +957,7 @@ table::stop() {
     }));
     _cache.refresh_snapshot();
 }
-
+static seastar::metrics::label_instance node_table_metrics("__per_table", "node");
```

Comment:
Why this? and with such an odd name?

Reply:
A name that starts with a double underscore is not reported. Having that extra label will allow an easy manipulation in run-time/load-time of those metrics, using the metrics relabel functionality.

An example of such a use case: report only specific keyspace or specific, table metric. 

In the future, I would like to reuse the label to mark the per-shard table metrics ```_per_table=shard```.